server/services/EmailService.js

```diff
@@ -0,0 +1,42 @@
+var nodemailer = require('nodemailer');
+
+
+
+class EmailService {
+  static sendMail(req, _, next){
+
+    const{email}=req.body;
+    var transporter = nodemailer.createTransport({
+        host: 'smtp.gmail.com',
+        port: 587,
+        secure: false,
+        requireTLS: true,
+        auth: {
+          user: process.env.Email_UserName,
+          pass: process.env.Email_Password
+        }
+      });
+      
+      
+      var mailOptions = {
+        from: process.env.Email_UserName,
+        to: email,
+        subject: process.env.Welcome,
+        html: process.env.Message
```

Comment:
Use email templates instead @Perry-code-luigi

Reply:
[Check here for Reference Blog](https://blog.stvmlbrn.com/2018/09/06/send-html-email-in-node.html)